external/ntpclient/Kconfig.protocol

```diff
@@ -25,6 +25,18 @@ config NETUTILS_NTPCLIENT_SIGWAKEUP
 	default 18
 	depends on !DISABLE_SIGNALS
 
+config NETUTILS_NTPCLIENT_DEFAULT_SERVER
+	string "NTP client default server"
+	default "0.ubuntu.pool.ntp.org"
+
+config NETUTILS_NTPCLIENT_DEFAULT_SERVER_PORT
+	int "NTP client default server port"
+	default 123
+
+config NETUTILS_NTPCLIENT_DEFAULT_INTERVAL_SECONDS
+	int "NTP client default interval seconds"
+	default 60
```

Comment:
It seems to be too frequent in 60 seconds. How about once a day?

Reply:
The interval time has been modified to a 24-hour period.